### ISSUE TITLE
imx6ul-var-dart: Kernel warnings present with newer kernel 5.4.85

### ISSUE THREAD

**Issue by acostach** [balena-os/balena-variscite#155]
dmesg reports:

```
[  636.523174] Unhandled fault: external abort on non-linefetch (0x008) at 0x76ef7000
[  636.530755] pgd = 32d4d09d
[  636.533469] [76ef7000] *pgd=99df5835, *pte=000f0783, *ppte=000f0e33
[  646.554278] 8<--- cut here ---
[  646.557371] Unhandled fault: external abort on non-linefetch (0x008) at 0x76eb9000
[  646.564955] pgd = 34cc2e95
[  646.567668] [76eb9000] *pgd=8c180835, *pte=000f0783, *ppte=000f0e33
```

**Comment by rosswesleyporter** [comment]
Doing a self-service update to Supervisor 13.1.10 will resolve the issue. Keeping this issue open until Supervisor 13.1.10 is included in a production release of balenaOS for imx6ul-var-dart.

**Comment by acostach** [comment]
Fixed in production at 2.98.33+rev1
